src/UCTNode.h

```diff
@@ -88,8 +95,7 @@ class UCTNode {
     float m_score;
     float m_net_eval{0};  // Original net eval for this node (not children).
     std::atomic<double> m_blackevals{0};
-    // node alive (not superko)
-    std::atomic<bool> m_valid{true};
+    std::atomic<Status> m_status{ACTIVE};
```

Comment:
pop quiz: What is `sizeof(Status)`?

Reply:
Good question. For me, I've been out of C++ programming for too long, but I imagine it's some form of standard integer by default. Therefore, I'll guess 4 bytes, maybe 8? Could easily be reduced to a single byte, I imagine. Or, if bit fields are possible (I can't remember) it would only require 2 bits. If booleans are only 1 bit, you could code it as two booleans. Do data structures have to conform to integer number of bytes? I forget.

Reply:
That does replace a 1-byte variable with a 4-byte one, but sizeof(UCTNode) stays unchanged.

Reply:
In UCTSearch.h for passflag_t GCP created his own constants, but I really don't like that code (and might do clean it up right now)

You can define the enum as char
http://en.cppreference.com/w/cpp/language/enum

Reply:
Given that it doesn't affect the total size of UCTNode, not having the underlying type be of the native CPU word size would more likely be detrimental than beneficial to performance.

Reply:
@TFiFiE: Since I've been out of the C++ loop (no pun intended) for so long, could you briefly (very briefly, not to bother you) explain why the total sizeof(UCTNode) doesn't change? Is it because data structures are already allocated in multiples of 4 bytes (words), and UCTNode was already 4k + 1 bytes in size, so increasing to 4k + 4 doesn't actually require an extra 4 byte word, since 4k + 1 already must be rounded up to 4k + 4 _[which = 4(k + 1) ]_? Just want to check my understanding; nothing super-important. Thanks. :-)

Reply:
see https://www.cprogramming.com/tutorial/size_of_class_object.html